.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
```

Comment:
I ran `npm test` and noticed there's some console logs left over in the older tests. Can we remove those here?

Reply:
Done.